controllers/apply/awards-for-all/fields.js

```diff
@@ -653,17 +746,9 @@ module.exports = function fieldsFor({ locale, data = {} }) {
             },
             get explanation() {
                 return localise({
-                    en: `<p>The start date needs to be at least ${
-                        this.settings.minStart.amount
-                    } ${
-                        this.settings.minStart.units
-                    } from when you plan to submit your application. If your project is a one-off event, please tell us the date of the event.</p>
-                    <p><strong>For example: ${
-                        this.settings.minDateExample
-                    }</strong>.</p>
-                    <p>The project end date must be within ${
-                        this.settings.maxDurationFromStart.label
-                    }
+                    en: `<p>The start date needs to be at least ${this.settings.minStart.amount} ${this.settings.minStart.units} from when you plan to submit your application. If your project is a one-off event, please tell us the date of the event.</p>
```

Comment:
We possibly have some sort of discrepancy between our editorconfigs here because these changes got made automatically when I saved...?!